src/main/java/edu/harvard/iq/dataverse/settings/FeatureFlags.java

```diff
@@ -30,12 +30,6 @@ public enum FeatureFlags {
      * @since Dataverse 5.14
      */
     API_SESSION_AUTH("api-session-auth"),
```

Comment:
When does api-session-auth go away? 🤔

Reply:
I think it is introduced for the SPA by @GPortas ?
I am not sure how efficient is the https://docs.payara.fish/enterprise/docs/Technical%20Documentation/Public%20API/OpenID%20Connect%20Support.html, but I would try not to implement to much ourselves.

Reply:
`api-session-auth` was always intended to be temporary, something we would get rid of, due to security concerns, as soon as proper auth for the SPA has been added.

Yes, it was added by @GPortas. Please see this PR:

- #9290